core/src/test/scalajvm/sttp/client4/testing/TestResponse.scala

```diff
@@ -0,0 +1,34 @@
+package sttp.client4.testing
+
+import sttp.client4.Response
+import sttp.model._
+
+private[sttp] object TestResponse {
```

Comment:
maybe let's call it `ResponseStub`, keeping the terminology consistent with `BackendStub`? Also, this should be public?

Reply:
yeah good idea with naming, as for public hmmm not sure if we want it to be expose to users ?

Reply:
yeah we do, it was exposed so far, and the idea is to use it when defining the behavior of a backend stub